proxy/proxy.go

```diff
@@ -972,6 +972,8 @@ func (p *Proxy) getRoundTripper(ctx *context, req *http.Request) (http.RoundTrip
 		f := "index.php"
 		if sf, ok := ctx.StateBag()["fastCgiFilename"]; ok {
 			f = sf.(string)
+		} else if req.URL.Path != "/" {
```

Comment:
`len(req.URL.Path) > 1 && req.URL.Path != "/"`

Reply:
I could not create empty path via
```
$ bin/skipper -inline-routes='* -> setRequestHeader("X-Path", "${request.path}") -> logHeader() -> <shunt>'
```

Reply:
done, Thanks.